common/logisticspipes/gui/guidebook/GuiGuideBook.java

```diff
@@ -0,0 +1,779 @@
+package logisticspipes.gui.guidebook;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.gui.FontRenderer;
+import net.minecraft.client.gui.GuiButton;
+import net.minecraft.client.gui.GuiScreen;
+import net.minecraft.client.renderer.BufferBuilder;
+import net.minecraft.client.renderer.GlStateManager;
+import net.minecraft.client.renderer.Tessellator;
+import net.minecraft.client.renderer.vertex.DefaultVertexFormats;
+import net.minecraft.item.ItemStack;
+import net.minecraft.nbt.NBTTagCompound;
+import net.minecraft.util.EnumHand;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.util.math.MathHelper;
+
+import logisticspipes.items.ItemGuideBook;
+import logisticspipes.utils.string.StringUtils;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.lwjgl.opengl.GL11;
+
+import logisticspipes.LPConstants;
+import logisticspipes.gui.guidebook.book.MenuItem;
+import logisticspipes.utils.GuideBookContents;
+
+public class GuiGuideBook extends GuiScreen {
```

Comment:
Would you be okay with this being licensed under MIT as well? If yes, please move it to the `network.rs485...` package then and have the default copyright generated (you should replace RS485 with your name of course).

Reply:
Could you explain whats the point? I don't mind if it is licensed under MIT but I don't understand the package change

Reply:
Added to the Glo Board